docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,370 @@
-lcd.DisplayStringAt(0, LINE(5), (uint8_t *)"Heard up", CENTER_MODE);  
-} else if(strcmp(found_command, "down") == 0) {  
-lcd.Clear(0xFFDB4437);  
-lcd.DisplayStringAt(0, LINE(5), (uint8_t *)"Heard down", CENTER_MODE);  
-} else if(strcmp(found_command, "unknown") == 0) {  
-lcd.Clear(0xFFF4B400);  
-lcd.DisplayStringAt(0, LINE(5), (uint8_t *)"Heard unknown", CENTER_MODE);  
-} else {  
-lcd.Clear(0xFF4285F4);  
-lcd.DisplayStringAt(0, LINE(5), (uint8_t *)"Heard silence", CENTER_MODE);  
-}  
-}  
-}
-```
-Now that we have updated the code, go back to the mbed directory:
-```
-cd <path_to_tensorflow>/tensorflow/lite/micro/tools/make/gen/mbed_cortex-m4/prj/micro_speech/mbed
-```
-and run the following command to rebuild the project:
-
-```mbed compile -m K66F -t GCC_ARM```
-
-Finally, copy the binary to the USB storage of the device, using the same method that you used earlier. You should now be able to say “up” and “down” to update the display.
-
-# Troubleshooting
+## Troubleshooting
 
 We have found some common errors that users face and have listed them here to help you get started with your application as quickly as possible.
-If you encounter:
-
-```Mbed CLI issues or Error: collect2: error: ld returned 1 exit status```
-
-Purge the cache with the following command:
-
-```mbed cache purge```
-
-You probably also have a stale BUILD folder. Clean up your directory and try again:
 
-```rm -rf BUILD```
+- If you encounter:
 
-Error: Prompt wrapping around line
+    ```
+    Mbed CLI issues or Error: collect2: error: ld returned 1 exit status
+    ```
 
-If your terminal is wrapping your text as show here:
+    Purge the cache with the following command:
 
-![Error prompt wrapping around line image](https://raw.githubusercontent.com/COTASPAR/K66F/master/images/troubleshooting.png)
+    ```
+    mbed cache purge
+    ```
 
-In your terminal type:
+    You probably also have a stale BUILD folder. Clean up your directory and try again:
 
-```export PS1='\u@\h: '```
+    ```
+    rm -rf BUILD
+    ```
 
-For a more minimalist type:
+- If your terminal is wrapping your text as show here:
 
-```export PS1='> '```
+    <span class="images">![Error prompt wrapping around line](https://raw.githubusercontent.com/COTASPAR/K66F/master/images/troubleshooting.png)</span>
 
-Error: "Requires make version 3.82 or later (current is 3.81)"
+    In your terminal, type:
 
-If you encounter this error, install the brew and make by typing the following code:
+    ```
+    export PS1='\u@\h: '
+    ```
 
-```
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-```
+    For a more minimalist type:
 
-```
-brew install make
-```
+    ```
+    export PS1='> '
+    ```
 
-**Note**: On a Mac, you might have to use gmake instead of make, to run your commands.
+- Error: `Requires make version 3.82 or later (current is 3.81)`
 
-Error: -bash: mbed: command not found
+    If you encounter this error, install Brew and Make:
```

Comment:
Assuming you're on a Mac

Reply:
In the interest of getting this published, we could remove the below:

> Error: Requires make version 3.82 or later (current is 3.81)
> 
> If you encounter this error, install Brew and Make:
> 
> ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
> 
> brew install make

Reply:
(or you could say "if you're using macOS")

Reply:
we could, but not sure this is extremely relevant as this was an error that used to occur a while back, it might be obsolete by now

Reply:
OK, worth removing altogether then!